src/js/disclosure/disclosure-view.js

```diff
@@ -8,7 +8,7 @@
  *
  * @module DisclosureView
  */
-;(function(define) {
+(function(define) {
```

Comment:
Is the starting semicolon not allowed? I think we use this extensively in edx-platform, but maybe this is a good incentive for us to remove it.